lsp/src/tree_sitter.rs

```diff
@@ -51,28 +53,58 @@ fn create_attribute(node: Node<'_>, source: &str) -> Option<Position> {
                 todo!("should fix this issue");
             }
         }
+
+        "fragment" | "element" => {
```

Comment:
I think I figured out a way to use [tree sitter query features](https://tree-sitter.github.io/tree-sitter/using-parsers#pattern-matching-with-queries) for this instead